conn_test.go

```diff
@@ -57,13 +58,17 @@ func TestJoinHostPort(t *testing.T) {
 	}
 }
 
-func testCluster(addr string, proto protoVersion) *ClusterConfig {
-	cluster := NewCluster(addr)
+func testMultiNodeCluster(addresses []string, proto protoVersion) *ClusterConfig {
```

Comment:
Define this like 

```go
func testMultiNodeCluster(proto protoVersion, addresses ...string) *ClusterConfig
```

That way callers dont need to wrap a single addr in a slice

Reply:
wasn't sure that it's ok to change the order, but will do now :-)